internal/versions/versions.go

```diff
@@ -168,16 +168,6 @@ func (h *handler) GetDefaultReleaseImage(cpuArchitecture string) (*models.Releas
 
 // Returns the OsImage entity
 func (h *handler) GetOsImage(openshiftVersion, cpuArchitecture string) (*models.OsImage, error) {
-	// For multiarch clusters we are offloading the validation checking for existing OS image matching the version
-	// and the architecture directly to the InfraEnv. This is because Cluster object on its own cannot know which
-	// architecture is going to be used and we cannot enforce that OS images exist for every architecture from the
-	// multiarch release image.
-	// In the multiarch scenario it's up to the InfraEnv's ISO generation part to detect whether the required
-	// OS image exists or not. This is because we still have InfraEnvs per-architecture and don't allow for multiarch
-	// InfraEnvs. Based on that, it's reasonable to skip the validation for Cluster and let InfraEnv handle it.
-	if cpuArchitecture == common.MultiCPUArchitecture {
```

Comment:
Is it safe to remove this now? No scenario calls GetOsImage with common.MultiCPUArchitecture?

Reply:
Yeah, no one should ever call this (because OS Image is function of InfraEnv). If by any reason it happens though, we will not fail, we will just return an error saying `The requested CPU architecture (multiarch) isn't specified in OS images list`